ui/app/styles/helper-classes/flexbox-and-grid.scss

```diff
@@ -104,6 +104,28 @@
   flex: 50%;
 }
 
+// moving away from !important, fresh flex styles below
```

Comment:
All of our flex classes are kind of specific and I really want to move away from using `!important` so opted to make more general styling here instead of things like `is-flex-between`

Reply:
+1 on moving away from `!important` and love the general styling!